libs/wingsdk/src/target-tf-aws/endpoint.ts

```diff
@@ -0,0 +1,48 @@
+import { TerraformOutput } from "cdktf";
+import { Construct } from "constructs";
+import { core } from "..";
+import * as cloud from "../cloud";
+import { CaseConventions, ResourceNames } from "../shared/resource-names";
+import { IInflightHost } from "../std";
+
+/**
+ * AWS implementation of `cloud.Endpoint`.
+ */
+export class Endpoint extends cloud.Endpoint {
+  constructor(scope: Construct, id: string, url: string) {
+    super(scope, id, url);
+
+    new TerraformOutput(this, "Url", {
+      value: this._url,
```

Comment:
```suggestion
      value: this.url,
```

Any reason to access the internal?